clusterloader2/api/cluster.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+// ClusterConfig is a structure that represents cluster description.
+type ClusterConfig struct {
+	KubeConfigPath string `json: kubeConfigPath`
+	Provider       string `json: provider`
+	Host           string `json: host`
```

Comment:
If you're not using any variables at the moment, just remove them. You can add later as needed,

Reply:
I've removed all fields except `KubeConfigPath` and `NodeCount`. The other fields will be needed when scheduler latency measurement is introduced.

Reply:
Sounds good. Do we even need NodeCount right now?

Reply:
Api latency measurement uses node count. If node count is provided by flag, explicit measurement parameter can be removed from `density/config.yaml`.

Reply:
Ah, you're right. We're using it for varying SLO for large clusters. However, given that we're going to move to the [new SLO definition](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md#steady-state-slisslos) which is independent of #nodes, we can probably exclude this.